### PR TITLE
New/xcrysden

### PR DESCRIPTION
**Pull request by shahzebsiddiqui**
@fcannini  creating this PR based on your comment in https://github.com/spack/spack/issues/17199#issuecomment-734500898.

### CODE REVIEW

**Comment by fcannini** [comment]
@shahzebsiddiqui Hi there!
I've changed jobs and am no longer involved in any spack-related activity. Please feel free to adopt this package.

**Comment by shahzebsiddiqui** [comment]
No worries @fcannini. I opened the PR so we dont lose this work. I'd prefer to get someone from xcrysden to help. I am not familiar with this package to be a maintainer.

**Comment by alalazo** [comment]
@fcannini Feel free to remove yourself from the list of maintainers if you don't want to receive notifications from @spackbot

File: var/spack/repos/builtin/packages/tcl-bwidget/package.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/tcl-bwidget/package.py:1]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/tcl-bwidget/package.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class TclBwidget(Package):
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/tcl-bwidget/package.py:9]: All of the current packages that extend `tcl` inherit from `AutotoolsPackage`.  Is this something to consider?

Also, several packages, including `tcl`, `tclap`, and `tk`, "inherit" from `SourceforgePackage` (actually a `mixin`) for the benefits of setting the url and mirrors.  

Have you considered at least taking advantage of the mixin?

File: var/spack/repos/builtin/packages/togl/package.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/togl/package.py:1]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/togl/package.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+from spack import *
+import os
+
+
+class Togl(AutotoolsPackage):
+    """ A Tcl/Tk widget for OpenGL rendering."""
+
+    homepage = "https://sourceforge.net/projects/togl"
+    url      = homepage + "/files/Togl/2.0/Togl2.0-src.tar.gz"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/togl/package.py:14]: Same suggestion here about considering the `SourceforgePackage` mixin.

File: var/spack/repos/builtin/packages/xcrysden/package.py [hunk 5]
```diff
@@ -0,0 +1,107 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/xcrysden/package.py:1]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]
At a minimum, the copyright years need to be updated.

Also suggested considering at least using the `SourceforgePackage` mixin for setting the url and mirrors.

**Comment by alalazo** [comment]
@shahzebsiddiqui Any update on this PR?

**Comment by alecbcs** [comment]
Going to go ahead and close this PR due to a lack of response. @shahzebsiddiqui happy to reopen if you're still working on this.

**Pull request closed without merging**